README.md

```diff
@@ -48,108 +49,104 @@ Remove all the completed todos after the `Clear completed` button click:
 - in case of any error show error message but process success deletions;
 
 ## If you want to enable tests
+
 - open `cypress/integration/page.spec.js`
 - replace `describe.skip` with `describe` for the root `describe`
 
 ## Instructions
+
 - Install Prettier Extention and use this [VSCode settings](https://mate-academy.github.io/fe-program/tools/vscode/settings.json) to enable format on save.
 - Implement a solution following the [React task guideline](https://github.com/mate-academy/react_task-guideline#react-tasks-guideline).
 - Use the [React TypeScript cheat sheet](https://mate-academy.github.io/fe-program/js/extra/react-typescript).
-- Replace `<your_account>` with your Github username in the [DEMO LINK](https://<your_account>.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
+- Replace `<your_account>` with your Github username in the [DEMO LINK](https://romanyshyns.github.io/react_todo-app-add-and-delete/) and add it to the PR description.
```

Comment:
Add demo link to the PR description 
<img width="939" alt="image" src="https://github.com/user-attachments/assets/3471f023-8c54-4b89-b639-0d562694d8ca">